setup.py

```diff
@@ -74,6 +74,7 @@ def _get_dbt_core_version():
         'google-cloud-bigquery>=1.25.0,<3',
         'google-api-core>=1.16.0,<3',
         'googleapis-common-protos>=1.6.0,<2',
+        'bigquery_schema_generator==1.5'
```

Comment:
Do we need to pin this to a specific version?

Reply:
Done! 1df7b45e23337c2ab12c474f5b43f78003a9db32